### DISCUSSION TITLE
Samsung ELPM482-00005 BMS compatability

### DISCUSSION THREAD

**Discussion by mattthemadscientist** [ai-republic/bms-to-inverter#69] [Q&A] [answered]
I'm hoping to get these batteries working with victron solar charge controllers and selectronic inverters. I've included an installation manual that has the communication protocol from the Samsung BMS, but I am unsure if this is the same as the "Samsung BMS" recently added to bms-to-inverter. 

key components of the BMS are listed as:
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/Matt/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/Matt/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72" lang=EN-AU style='tab-interval:36pt'>

MCU | STM32F105VCT6 | -
-- | -- | --
Current Sensing IC | R2A24060 | -
AFE | ML5236 | 14Ch
Shunt Resistor | PSR500HTQFH1L00 | 1mΩ, 1%, 5W, 2   Parallel
2nd protection fuse | BZ05-640 | 75A, 64Ω
FET | FDB019N807L | Rated 270A, 80Vdc 5 Parallel
Fuse | 250GH-63UL | 250V, 63A

</body>

</html>

Is this the same architecture that the Samsung BMS has already supported? there are quite a few of these batteries becoming available here in Australia, and they are nearing the end of their storage life if we don't come up with a solution to make sure we can utilise them. 

I'll have to become a sponsor I think :-) this is an amazing project and you have done more for the environment by developing this than many do in a lifetime. 

[4.84kWh_Standard_Product_Specification(1)[92].pdf](https://github.com/user-attachments/files/17270963/4.84kWh_Standard_Product_Specification.1.92.pdf)

**Comment by ai-republic** [comment, marked as answer]
Hi @mattthemadscientist,
this is matches exactly the _SAMSUNG_BMS_ binding that I already implemented.
You don't _have_ to be a sponsor to get reply 😆 but any help is very much appreciated 😉
